validator/src/cli.rs

```diff
@@ -906,6 +907,60 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .hidden(hidden_unless_forced())
                 .help("Controls if to use QUIC to send votes."),
         )
+        .arg(
+            Arg::with_name("tpu_max_connections_per_peer")
+                .long("tpu-max-connections-per-peer")
+                .takes_value(true)
+                .default_value(&default_args.tpu_max_connections_per_peer)
+                .validator(is_parsable::<u32>)
+                .hidden(hidden_unless_forced())
+                .help("Controls the max concurrent connections per IpAddr."),
+        )
+        .arg(
+            Arg::with_name("max_tpu_staked_connections")
```

Comment:
we should be consistent and make all the CLIs tpu_max_ instead of having some
max_tpu_ and some tpu_max_

Reply:
done